Files/Strings/en-US/Resources.resw

```diff
@@ -2791,6 +2791,9 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="SettingsImportErrorTitle" xml:space="preserve">
     <value>Error importing settings</value>
   </data>
+  <data name="InnerEmptyRecycleBinButton.Label" xml:space="preserve">
```

Comment:
```suggestion
  <data name="Empty" xml:space="preserve">
```
For context https://devblogs.microsoft.com/ifdef-windows/use-a-custom-resource-markup-extension-to-succeed-at-ui-string-globalization/